pkg/domain/infra/abi/system.go

```diff
@@ -123,9 +123,10 @@ func (ic *ContainerEngine) SetupRootless(cmd *cobra.Command) error {
 		}
 	}
 
-	if !executeCommandInUserNS(cmd) {
-		return nil
-	}
+	// Check moved into presentation layer
```

Comment:
Why not delete?

Reply:
I wanted to give a hint where it went. :-) I'll remove it with next push.